### DISCUSSION TITLE
sendButtons in N8N

### DISCUSSION THREAD

**Discussion by 2tmmkt** [devlikeapro/waha#693] [Q&A]
I am trying to send buttons with WAHA IN n8n but it's not working. I get a output saying that it was sent but the status is "PENDING" and it's not delivered in WhatsApp.

Is this feature still running? Has anyone used it in N8N?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! Yeah, buttons are fragily creatures, it gets deprecated fast on WA side :( Looking for a new solution for that! :crossed_fingers: 
Consider using "send 1 back to {action}" regular text message type for now

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
